.travis.yml

```diff
@@ -0,0 +1,41 @@
+---
+sudo: required
+
+language: python
+python:
+  - 3.6
+matrix:
+  include:
+    - python: 3.7
+      dist: xenial
+      sudo: true
+
+branches:
+  only:
+    - /unreviewed.*/
+    - /devel.*/
+
+env:
+  global:
+    # yamllint disable-line rule:line-length
```

Comment:
Right now, some issues appeared related to secure the credentials in a encrypted way in travis. This method worked in my fork but on this repo this has failed. The credentials, in order for the build to pass, has been stored in travis console but this needs to be reviewed in a close future.

Reply:
@dmonteiro-paddle can we do the travis work in a separate PR;  I'd like this to cover only the changes that were done for the previous deploy + the changes to fix the things shippable broke.

If there isn't something else, please approve and please put in a PR for the travis changes.  We should then move this over into the other repo as well.